main.py

```diff
@@ -89,7 +89,7 @@ def train(
         model_save_path: 'Where to export the completed generation.'):
     print("Training on:", tf_records[0], "to", tf_records[-1])
     with utils.logged_timer("Training"):
-        dual_net.train(working_dir, tf_records)
+        dual_net.train(working_dir, *tf_records)
```

Comment:
you'll need to update rl_loop for this as well.

Does local_rl_loop run?  that's no longer in tests.sh or run_tests.py after the flags work...

Reply:
rl_loop should be fine for now.

Reply:
local_rl_loop still runs fine, there's a lot of deferred flag-fixup work... :(

Reply:
rl_loop calls main.train, and the *expansion shouldn't work as it looks right now, IIUC